CHANGELOG.md

```diff
@@ -1,5 +1,91 @@
+```
+
+When you use programming API, pass HTTPS server options to the [createTestCafe](https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/createtestcafe.html) method.
+
+```js
+'use strict';
+
+const createTestCafe        = require('testcafe');
+const selfSignedSertificate = require('openssl-self-signed-certificate');
+let runner                  = null;
+
+const sslOptions = {
+    key:  selfSignedSertificate.key,
+    cert: selfSignedSertificate.cert
+};
+
+createTestCafe('localhost', 1337, 1338, sslOptions)
+    .then(testcafe => {
+        runner = testcafe.createRunner();
+    })
+    .then(() => {
+        return runner
+            .src('test.js')
+
+            // Browsers restrict self-signed certificate usage unless you
+            // explicitly set a flag specific to each browser.
+            // For Chrome, this is '--allow-insecure-localhost'.
+            .browsers('chrome --allow-insecure-localhost')
+            .run();
+    });
+```
+
+See [Connect to TestCafe Server over HTTPS](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/connect-to-the-testcafe-server-over-https.html) for more information.
+
+#### :gear: Construct Screenshot Paths with Patterns ([#2152](https://github.com/DevExpress/testcafe/issues/2152))
+
+You can now use patterns to construct paths to screenshots. TestCafe provides a number of variables you can include in the path. For instance, these are `${DATE}`, `${TIME}`, `${USERAGENT}`, etc. For a complete list, refer to the command line [--screenshot-path-pattern flag description](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#-p---screenshot-path-pattern).
+
+You specify a screenshot path pattern when you run tests. Each time TestCafe takes a screenshot, it substitutes the variables with their values and saves the screenshot to the resulting path.
+
+The following example shows how to specify a screenshot path pattern through the command line.
+
+```sh
+testcafe all test.js -s screenshots -p "${DATE}_${TIME}/test-${TEST_INDEX}/${USERAGENT}/${FILE_INDEX}.png"
+```
+
+When you use programming API, pass the screenshot path pattern to the [runner.screenshots method](https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/runner.html#screenshots).
+
+```js
+runner.screenshots('reports/screenshots/', true, '${TEST_INDEX}/${OS}/${BROWSER}-v${BROWSER_VERSION}/${FILE_INDEX}.png');
+```
+
+#### :gear: Add Info About Screenshots and Quarantine Attempts to Custom Reports ([#2216](https://github.com/DevExpress/testcafe/issues/2216))
+
+Custom reporters can now access data about screenshots and the history of quarantine attempts (if the test ran in the quarantine mode).
+
+The following info about screenshots is now available:
+
+* the path to the screenshot file,
+* the path to the thumbnail image,
+* the user agent in which the screenshot was taken,
```

Comment:
I remember that we discuss about it, but we cannot take screenshot in the user agent.
This should be more clear
```js
the user agent of the browser in which the screenshot was taken
```